src/main/scala/org/globalforestwatch/layers/TotalCarbonEmisYear.scala

```diff
@@ -5,6 +5,8 @@ import org.globalforestwatch.grids.GridTile
 case class TotalCarbonEmisYear(gridTile: GridTile, model: String="standard")
   extends FloatLayer
     with OptionalFLayer {
-  val model_suffix = if (model == "standard") "" else s"__$model"
-  val uri: String = s"$basePath/gfw_total_carbon_stock_in_emissions_year$model_suffix/v20190816/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/is/geotiff/${gridTile.tileId}.tif"
```

Comment:
You can delete this path from the datalake. It's not actually used in Geotrellis.